js/bundles/modules/file_providers.js

```diff
@@ -1,8 +1,6 @@
 import DevExpress from "./core";
 import WebApiFileProvider from "../../ui/file_manager/file_provider/webapi";
-import OneDriveFileProvider from "../../ui/file_manager/file_provider/onedrive";
```

Comment:
Why didn't you remove the OneDriveFileProvider file?

Reply:
Removed